src/applications/claims-status/tests/components/FilesOptional.unit.spec.jsx

```diff
@@ -19,8 +19,10 @@ describe('<FilesOptionalOld>', () => {
 
     getByText(item.displayName);
     getByText(item.description);
-    getByText('Optional');
-    getByText('- We requested this from others, but upload it if you have it.');
+    getByText(
+      "You don't have to do anything, but if you have this information you can",
```

Comment:
I used double quotes here because the linter was erroring when I used single quotes with an `\'` to escape the single quote